R/Result.R

```diff
@@ -9,11 +9,11 @@ NULL
 #' @docType methods
 NULL
 
-OdbcResult <- function(connection, statement, params = NULL, immediate = FALSE) {
+OdbcResult <- function(connection, statement, params = NULL, immediate = FALSE, query_timeout = 0L) {
```

Comment:
better keep the default value exactly the same.

```suggestion
OdbcResult <- function(connection, statement, params = NULL, immediate = FALSE, query_timeout = 0) {
```